### ISSUE TITLE
[RA2 Ch2] Requirements calls for multi-tenancy which Kubernetes doesn't support

### ISSUE THREAD

**Issue by fkautz** [anuket-project/anuket-specifications#1958]
Kubernetes does not support multitenancy but does support "hierarchical namespaces," which isn't multi-tenancy as we think of it. Global resources and configurations are co-mingled in K8s.

The side effect of isolation that multi-tenancy gives you can be achieved through Kubernetes policy, but it isn't "multi-tenancy." However, an application hosted on Kubernetes can support multi-tenancy, but this is a feature of the application being hosted rather than Kubernetes itself. 

In short, we have multi-tenancy requirements in RA2, which Kubernetes itself will be unable to meet. If this is a hard requirement, we'll need to spin up separate K8s clusters.

**Comment by petorre** [comment]
Similar text is already in 6.2.2 and 6.3.2. Multi-tenancy and workload isolation with Kubernetes.
